src/avram/tasks/db/drop.cr

```diff
@@ -3,6 +3,9 @@ require "colorize"
 class Db::Drop < BaseTask
   summary "Drop the database"
 
+  def initialize(@quiet : Bool = false)
```

Comment:
Similar to the [create_db](https://github.com/luckyframework/avram/blob/master/src/avram/migrator/runner.cr#L67) method, this variable will need to be passed in to the `drop_db` method in the `run_task` method below.